privacyidea/static/components/user/factories/user.js

```diff
@@ -87,7 +87,7 @@ myApp.factory("UserFactory", ['AuthFactory', '$http', '$state', '$rootScope',
                     function(error) {AuthFactory.authError(error.data)});
             },
             deleteCustomAttribute: function(username, realmname, key, callback) {
-                $http.delete(userUrl + "/attribute/" + key + "/" + username + "/" + realmname,
+                $http.delete(userUrl + "/attribute/" + encodeURIComponent(key) + "/" + encodeURIComponent(username) + "/" + encodeURIComponent(realmname),
```

Comment:
Same as above for the `realmname`.

Reply:
proposed changes implemented in [bfc9af4](https://github.com/privacyidea/privacyidea/pull/3306/commits/bfc9af4ac9a8d357b4fa1e74452474bf89a60d74)